app/src/main/java/com/example/bottomnav/Parse.java

```diff
@@ -32,6 +31,7 @@ public Parse(char[] code) throws Exception {
         IASTTranslationUnit translationUnit = getIASTTranslationUnit(code);
         IASTNode[] comments = translationUnit.getComments();
         IASTNode[] children = translationUnit.getChildren();
+        HashMap<String, Integer> NameToId = new HashMap<>();
```

Comment:
```suggestion
        HashMap<String, Integer> NameToId = new HashMap<>();
```

similarly, consistent variable naming style?